interface/forms/vitals/vitals.js

```diff
@@ -57,12 +57,18 @@
         let value = node.value;
         let inputSave = document.getElementById(targetSaveUnit);
         if (!inputSave) {
-            console.error("Failed to find node with data-target-input of ", targetSaveUnit);
+            console.error(
```

Comment:
Can we remove the style changes to the vitals.js file.

Reply:
Removed the styling/formatting changes.